scalafix-tests/unit/src/test/scala/scalafix/tests/util/IntervalSetSpec.scala

```diff
@@ -0,0 +1,46 @@
+package scalafix.tests.util
+
+import org.scalatest.FunSuite
+import org.scalactic.source.Position
+
+import scalafix.internal.util.IntervalSet
+
+class IntervalSetSpec() extends FunSuite {
```

Comment:
IntervalSetTests to follow convention. I may soon rename `s/Tests/Suite/` but for now let's try to keep it consistent.